examples/vision/bit.py

```diff
@@ -0,0 +1,322 @@
+"""
+Title: Image Classification using BigTransfer(BiT)
+Author: [Sayan Nath](https://twitter.com/sayannath2350)
+Date created: 2021/08/29 
+Last modified: 2021/08/29
+Description: BigTransfer (BiT) State-of-the-art transfer learning for image classification.
+"""
+
+"""
+## Introduction
+
+
+BigTransfer known as BiT is a State-of-the-art transfer learning method for Image Classification Tasks. a set of pre-trained image models that can be transferred to obtain excellent performance on new datasets, even with only a few examples per class. BiT performs well across a surprisingly wide range of data regimes — from 1 example per class to 1M total examples. A detailed analysis of the main components are described [here](https://arxiv.org/pdf/1912.11370.pdf) which leads to high transfer performance.
```

Comment:
Please make sure all text lines are less than 80 characters long, otherwise any review (or version control) is very difficult

Reply:
`
BigTransfer known as BiT is a State-of-the-art transfer learning method for Image Classification Tasks. A set of pre-trained image models that can be transferred to obtain excellent performance on new datasets, even with only a few examples per class. BiT performs well across a surprisingly wide range of data regimes — from 1 example per class to 1M total examples.
`

Reply:
Now update the PR to follow this rule :)